dev/TabView/TabView.h

```diff
@@ -26,6 +27,20 @@ class TabViewTabClosingEventArgs :
     winrt::IInspectable m_item{};
 };
 
+class TabViewTabDraggedOutsideEventArgs :
+    public winrt::implementation::TabViewTabDraggedOutsideEventArgsT<TabViewTabDraggedOutsideEventArgs>
+{
+public:
+    TabViewTabDraggedOutsideEventArgs(winrt::IInspectable const& item, winrt::TabViewItem tab) { m_item = item; m_tab = tab; }
+
+    winrt::IInspectable Item() { return m_item; }
+    winrt::TabViewItem Tab() { return m_tab; }
+
+private:
+    winrt::IInspectable m_item{};
```

Comment:
>winrt::IInspectable m_item{}; [](start = 4, length = 29)

should these be tracker_refs? I'm not sure.

Reply:
Yes, the tab should -- I'll add it.